### ISSUE TITLE
Raise error on allow(user, action, resource, undefined)

### ISSUE THREAD

**Issue by chrisjensen** [vadimdemedes/cancan#27]
If, due to a referencing mistake or similar, the condition is `undefined` or `null`, authorization will quietly pass without the condition.

I'd suggest that if it explicitly receives null it should throw an error, or at least log a warning to the console.

Suggest that `allow` does:
```
  allow(model, actions, targets, condition) {
    if (arguments.length > 3 && !condition) console.log("It looks like you've explicitly passed 'undefined' for the condition. Are you sure you wanted to do that? (allow(" + arguments + ")");
```

**Comment by vadimdemedes** [comment]
Pushed https://github.com/vadimdemedes/cancan/commit/f9efebb025386c1a33875478a5df2f5e1fefe14c to fix this, thanks for reporting!

**Comment by vadimdemedes** [comment]
Just published `3.0.1` with a fix.
